CombinatorialGames/IGame/IGame.lean

```diff
@@ -0,0 +1,529 @@
+/-
+Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import CombinatorialGames.Game.PGame
+import Mathlib.Algebra.Group.Pointwise.Set.Basic
+import Mathlib.Logic.Hydra
+import Mathlib.Logic.Small.Set
+
+universe u
+
+-- This is a false positive due to the provisional duplicated IGame/IGame file path.
+set_option linter.dupNamespace false
+-- All computation should be done through `IGame.Short`.
+noncomputable section
+
+open PGame Set Pointwise
+
+/-- Games up to identity.
+
+TODO: write proper docstring. -/
+def IGame : Type (u + 1) :=
+  Quotient identicalSetoid
+
+namespace IGame
+
+/-! ### Game moves -/
+
+/-- The quotient map from `PGame` into `IGame`. -/
+def mk (x : PGame) : IGame := Quotient.mk _ x
+theorem mk_eq_mk {x y : PGame} : mk x = mk y ↔ x ≡ y := Quotient.eq
+
+alias ⟨_, mk_eq⟩ := mk_eq_mk
+alias _root_.PGame.Identical.mk_eq := mk_eq
+
+@[cases_eliminator]
+theorem ind {P : IGame → Prop} (H : ∀ y, P (mk y)) (x : IGame) : P x :=
+  Quotient.ind H x
+
+/-- Choose an element of the equivalence class using the axiom of choice. -/
+def out (x : IGame) : PGame := Quotient.out x
+@[simp] theorem out_eq (x : IGame) : mk x.out = x := Quotient.out_eq x
+
+-- TODO: docstring
```

Comment:
```suggestion
/--
`Quotient.lift` for `IGame`: if f : PGame → α respects the identical equivalence relation ≡, then f
lifts IGame to α
-/
```

Reply:
On second consideration, I only use this definition twice, in order to define `IGame.leftMoves` and `IGame.rightMoves`. Since we want to expose `PGame` as little as possible within the API, I've simply inlined `Quotient.lift` into their definitions.